Mlem/App/Views/Shared/Images/Core/MediaView+Views.swift

```diff
@@ -126,7 +126,29 @@ extension MediaView {
                             .foregroundStyle(.themedAccent)
                             .buttonStyle(.bordered)
                         }
+                        
+                        if developerMode {
+                            DisclosureGroup("Details") {
+                                Text(error.localizedDescription)
+                                    .foregroundStyle(.themedNegative)
+                                    .multilineTextAlignment(.center)
+                                    .padding(.top)
+                                
+                                Button("Copy Error", icon: .general.copy) {
+                                    UIPasteboard.general.string = error.localizedDescription
+                                    ToastModel.main.add(.success(String("Copied")))
```

Comment:
We should consistently either localize or not localize these strings. We've localized the button label (`"Copy Error"`), so we should localize this too. Or, unlocalize both. I don't think there's much harm in localizing this because it's likely to appear elsewhere too.

```suggestion
                                    ToastModel.main.add(.success("Copied"))
```